src/IROperator.h

```diff
@@ -820,7 +885,12 @@ inline Expr pow(Expr x, Expr y) {
     if (x.type() == Float(64)) {
         y = cast<double>(y);
         return Internal::Call::make(Float(64), "pow_f64", {x, y}, Internal::Call::Extern);
-    } else {
+    }
+    else if (x.type() == Float(16)) {
+         y = cast<float16_t>(y);
+        return Internal::Call::make(Float(16), "pow_f16", {x, y}, Internal::Call::Extern);
```

Comment:
Oh, just noticed the commit comment. I guess whether or not these exist might be backend-specific, hrm... should we provide them for all backends, with a slow implementation as the default, or should we error out in the backend if we detect a float16 math intrinsic. I suppose backends that don't support float16 math will already have thrown an error.